tests/integration/test_charm.py

```diff
@@ -124,10 +125,47 @@ async def test_integration_with_s3_integrator(
     )
 
     # Assert that both kyuubi-k8s and s3-integrator charms are in active state
-    assert ops_test.model.applications[APP_NAME].status == "active"
+    assert ops_test.model.applications[APP_NAME].status == "blocked"
```

Comment:
suggestion: Also assert that the charm waits on "missing integration hub" message.

Reply:
Actually, we never do such a message check here (I just wanted to copy and paste that from elsewhere). I'm happy to do this, even in the next couple of days, but in a dedicated PR where we do this also in the other cases.

Reply:
[DPE-4697](https://warthogs.atlassian.net/browse/DPE-4697)